applications/crossbar/src/crossbar_view.erl

```diff
@@ -223,7 +224,6 @@ build_load_range_params(Context, View, Options) ->
 
             HasQSFilter = crossbar_filter:is_defined(Context)
                 andalso not crossbar_filter:is_only_time_filter(Context, TimeFilterKey),
-            lager:debug("has qs filter: ~s", [HasQSFilter]),
```

Comment:
Why removing this? I only prints one time.

Reply:
I was just deleting all the "has qs filter" logs since I had added a bunch throughout. I'll re-add this one